lib/ui/chat_screen.dart

```diff
@@ -11,7 +11,7 @@ class ChatScreen extends StatefulWidget {
 
 class _ChatScreenState extends State<ChatScreen> {
   var _repository = Repository();
-  User _user = User();
+  User user = User();                                           //fix was user instead of _User  
```

Comment:
User object is not needed in this class. You can remove this statement as well 👍